enums/eventBrandMembership.enum.go

```diff
@@ -0,0 +1,23 @@
+package enums
+
+type EventMembershipRole int
+
+const (
+	Admin EventMembershipRole = iota
+	Contributor
+	Attendee
+)
+
+func GetRoleDescription(role int) string {
```

Comment:
You can utilize the int type alias here:
```suggestion
func GetRoleDescription(role EventMembershipRole) string {
```